executor/distsql_test.go

```diff
@@ -581,3 +582,49 @@ func TestAdaptiveClosestRead(t *testing.T) {
 	// 2 IndexScan with cost 19/56, 2 TableReader with cost 32.5/65.
 	checkMetrics("select/* +USE_INDEX_MERGE(t) */ id from t use index(`idx_v_s1`) use index(idx_s2) where (s1 < 3 and v > 0) or s2 = 3;", 3, 1)
 }
+
+func TestCoprocessorPagingReqKeyRangeSorted(t *testing.T) {
```

Comment:
The added test can only cover (literally) the added code.
It can't prevent regression in the future when code changes.